packages/trigger-client/src/TriggerClient.ts

```diff
@@ -83,6 +83,7 @@ export class TriggerClient {
     );
 
     const executeParams: TriggerServerExecuteRequestByType['place_order'] = {
+      id: params.id?.toNumber() || null,
```

Comment:
We should be using `??` (can we adjust on engine if its not being used there?)

Reply:
oh ic, think we are not using there will fix. what's the diff again? 😭

Reply:
oh nvm we are!

Reply:
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or

Reply:
oh TIL!

Reply:
(actually you might have point this out in the past but think i won't forget again!)